modular_skyrat/modules/reagent_forging/code/forge_weapons.dm

```diff
@@ -62,15 +118,59 @@
 	. = ..()
 	AddComponent(/datum/component/two_handed, force_unwielded=8, force_wielded=16)
 
+/obj/item/forging/reagent_weapon/bokken
```

Comment:
Same as my last comment on the bokken; a bokken is, by definition, a wooden training sword, and should NOT be a forging product. If you're looking for a sword with a block chance, try a Rapier, which is designed to be used defensively in a duel, or a Sabre, who's handguard means that the hand is naturally blocked as well as the blade being narrow enough to maneuver/defend with.

But by all means add bokkens as a training weapon :) Just dont have them able to do much against real weapons please, we dont need a repeat of meteor-parrying.